spec/factories/state_file_az_intakes.rb

```diff
@@ -102,6 +102,7 @@
     raw_direct_file_data { File.read(Rails.root.join('app', 'controllers', 'state_file', 'questions', 'df_return_sample.xml')) }
     primary_first_name { "Ariz" }
     primary_last_name { "Onian" }
+    primary_birth_date { Date.new((MultiTenantService.statefile.current_tax_year - 65), 12, 1) }
```

Comment:
there were a few calculations that were changing across the codebase because they are impacted by the number of people (out of primary and spouse) that are 65+. Previously we were grabbing this number from the df data and now we are calculating it based on the birthdate we get since there are different rules for different states. 

The factory instance is using df_return_sample.xml which indicates that the primary is 65+ (Primary65OrOlderInd) but doesn't have a primary_birth_date. I added a 65+ birthdate here so that it would be consistent with the DF data used in this instance and to resolve the failing tests.